generate-release-notes.js

```diff
@@ -9,13 +9,15 @@ async function generateReleaseNotes({
 	range,
 	exclude = '',
 	commitTemplate = '- {hash} {title}',
-	releaseTemplate = '{commits}\n\n{range}'
+	releaseTemplate = '{commits}\n\n{range}',
+	dateFormat = '--date=short'
 }) {
 	// Get commits between computed range
-	let {stdout: commits} = await execFile('git', ['log', '--format=%H%s', range]);
+	let {stdout: commits} = await execFile('git', ['log', '--format=%H¬%ad¬%s', dateFormat, range]);
 	commits = commits.split('\n').filter(Boolean).map(line => ({
```

Comment:
Let’s only call split once.

```suggestion
	commits = commits.split('\n').filter(Boolean).map(line => {
	    const [hash, date, title] = line.split(etc)
```

Reply:
I tried it and it causes error – `TypeError: Right side of assignment cannot be destructured`
```js
commits = commits.split('\n').filter(Boolean).map(line => {
	const [hash, date, title] = line.split('¬'),
	hash.slice(0, 8)
});
```
I am newbie to JS and I don't know where is problem...

Reply:
You’re using a comma at the end of the line but it should be a semicolon